.github/workflows/nightly-pipeline.yaml

```
@@ -457,3 +458,38 @@ jobs:
           propagate_failure: false
           trigger_workflow: true
           wait_workflow: true
+  ucxx-build:
+      needs: [get-run-info, ucx-py-build, rmm-build]
```

```suggestion
      needs: [get-run-info, rmm-build]
```

`ucxx` doesn't depend on `ucx-py`.

I think you might be confusing `ucx` with `ucx-py`: https://github.com/rapidsai/ucxx/blob/abb9b62122d37ed7eef6bc2988c379ec06491440/conda/recipes/ucxx/meta.yaml#L166